src/Features/VisualBasic/Portable/Structure/Providers/MultiLineIfBlockStructureProvider.vb

```diff
@@ -14,9 +14,165 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Structure
                                                   spans As ArrayBuilder(Of BlockSpan),
                                                   options As OptionSet,
                                                   cancellationToken As CancellationToken)
-            spans.AddIfNotNull(CreateBlockSpanFromBlock(
-                               node, node.IfStatement, autoCollapse:=False,
-                               type:=BlockTypes.Conditional, isCollapsible:=True))
+
+            ' The full block: eg If ... Then ... End If
+            spans.AddIfNotNull(
+                CreateBlockSpanFromBlock(
+                                          node,
+                                          node.IfStatement,
+                            autoCollapse:=False,
+                                    type:=BlockTypes.Conditional,
+                           isCollapsible:=True))
```

Comment:
this alignment style does not match any of the IDE code.  Please follow IDE coding patterns.

Reply:
@CyrusNajmabadi This style is more clear and readable than the existing style.